AMANDA/amanda_to_s3.py

```diff
@@ -130,6 +130,23 @@
         AND f.WORKCODE = 50590
         AND fa.RESULTCODE = 61510
     """,
+    "ex_permits_issued": """
+    SELECT
+        CONCAT(CONCAT(f.FOLDERYEAR, '-'), f.FOLDERSEQUENCE) AS PERMIT_ID,
```

Comment:
TIL that Oracle DB concat requires nesting like this for more than 2 args. 👍

Reply:
maybe there is a non-nested way haha 😅 https://ramkedem.com/en/oracle-string-concatenation/#:~:text=Oracle%20String%20concatenation%20allows%20you,concatenation%20operator%20(%7C%7C).

Reply:
maybe there is a non-nested way haha 😅 https://ramkedem.com/en/oracle-string-concatenation/#:~:text=Oracle%20String%20concatenation%20allows%20you,concatenation%20operator%20(%7C%7C).

Reply:
Whoa - pipes! That is pretty cool